### PR TITLE
ARTEMIS-4680 - Upgrade the console to use HawtIO 4

### PR DESCRIPTION
**Pull request by andytaylor**
Added a select all to the browse page

fixed and added the rat report jobs

a couple of typos

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -26,5 +26,26 @@ jobs:
...
+        with:
+          name: rat-extension-${{ matrix.java }}
+          path: artemis-console-extension/target/rat.txt
+
+      - name: Upload Rat Report On Failure
+        if: failure()
+        uses: actions/upload-artifact@v4
+        with:
+          name: rat-console-${{ matrix.java }}
+          path: artemis-console-war/target/rat.txt
```
Comment (gemmellr) [inline .github/workflows/build.yml:51]: Having lots of steps doing essentially the same overall thing, and which are likely to individually fail themselves if the file doenst exist (job could fail for other reasons) seems less than ideal. The per-module nature of it is also brittle in terms of handling any future additions and renames.

Should be able to do it in one step, with one upload, and which doesnt fail if there are no rat.txt files.

https://github.com/actions/upload-artifact?tab=readme-ov-file#upload-using-a-wildcard-pattern
https://github.com/actions/upload-artifact?tab=readme-ov-file#customization-if-no-files-are-found

**Comment by jbertram** [comment]
FWIW, the select-all addition to the Browse tab looks good.

File: .github/workflows/build.yml (hunk 1 above)
Comment (andytaylor) [reply .github/workflows/build.yml:51]: I will fix this monday

Comment (andytaylor) [reply .github/workflows/build.yml]: fixed

**Pull request merged**